### ISSUE TITLE
Make generic run-scripts

### ISSUE THREAD

**Issue by uniomni** [uniomni/PyARR#5]
There are lots of runscripts that are almost identical. Work on a more data driven approach.
Also, the config file in this project should be split up into two:

One which is about real configuration - i.e. things that don't change often such as MyTimeStep or Cellsize or quantities.
Another one which is 'per run' and 'per user'

**Comment by Dr-Shoznum** [comment]
Yes that sounds like a really good idea. Even the storms durations and
blockage do not change. So a script that just looks to see what is
available and then processes whats there and skips what's not there.

**Comment by RudyFrom3** [comment]
I have some ideas of how to implement this... lets chat.... (As once a full generic script is completed... the ability to run ANUGA through things like QGIS become far easier to manage I think)...
